From c17760bffe3085efeea39df0e2109d831e19664d Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Wed, 16 Jun 2021 16:27:51 +0200
Subject: [PATCH] imx8mm-var-dart-nrt: Add workaround for meta-balena u-boot append not applied

The meta-balena u-boot append recipe does not apply to
this repository because it's named u-boot_%.bbappend
while the u-boot provided by Variscite is named
u-boot-variscite. Also, the production configs
aren't merged by meta-balena u-boot_%.bbappend
since it tests for UBOOT_CONFIG not being set,
and in this case UBOOT_CONFIG needs to be set
to "sd" for the imx-boot image to be generated.

We add this workaround for now here for an
urgent release and opened the issue no. 2217
in meta-balena to discuss what the right solution
would be for this cases.

Upstream-status: Inappropritate[configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 configs/imx8mm_var_dart_defconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/configs/imx8mm_var_dart_defconfig b/configs/imx8mm_var_dart_defconfig
index f79f9221c8..0e3dec9d15 100644
--- a/configs/imx8mm_var_dart_defconfig
+++ b/configs/imx8mm_var_dart_defconfig
@@ -69,3 +69,10 @@ CONFIG_CMD_EXT4=y
 CONFIG_CMD_FAT=y
 CONFIG_CMD_EXPORTENV=y
 CONFIG_DM_I2C=y
+CONFIG_BOOTDELAY=-2
+CONFIG_DISABLE_CONSOLE=y
+CONFIG_SILENT_CONSOLE=y
+CONFIG_SILENT_CONSOLE_UPDATE_ON_RELOC=y
+CONFIG_SYS_DEVICE_NULLDEV=y
+CONFIG_AUTOBOOT_KEYED=y
+CONFIG_AUTOBOOT_KEYED_CTRLC=n
-- 
2.17.1

